modules/core/lib/Auth/Process/PairwiseID.php

```diff
@@ -35,6 +35,20 @@
  */
 class PairwiseID extends Auth\ProcessingFilter
 {
+    /**
+     * The regular expression to match the identifier
+     *
+     * @var string
+     */
+    public const $subject_regex = '/^[a-zA-Z0-9]{1}[a-zA-Z0-9=-]{0,126}$/i';
```

Comment:
The `{1}` does not really add anything I guess. The `A-Z` is unneeded if you add the `i` on the end.